src/main/java/com/ecommerce/ECommerceApp/model/CartItem.java

```diff
@@ -2,6 +2,10 @@
 
 import org.springframework.stereotype.Component;
 
+/**
+ * Represents item in a shopping care.
+ * Contains productID, quantity, and price.
+ */
```

Comment:
Added missing Javadoc to help clarify purpose of code, keeping it maintainable and readable.

Source: https://www.tutorialspoint.com/java/java_documentation.htm